pkg/kubectl/cmd/patch.go

```diff
@@ -83,6 +88,18 @@ func RunPatch(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []stri
 		return err
 	}
 
+	patchType := api.StrategicMergePatchType
+	patchTypeString := strings.ToLower(cmdutil.GetFlagString(cmd, "type"))
+	if len(patchTypeString) == 0 {
+		patchType = api.StrategicMergePatchType
+	} else {
```

Comment:
Can you just make this line `if len(patchTypeString) != 0 {` and delete the two lines above it?